.travis.yml

```diff
@@ -5,7 +5,9 @@ d:
   - ldc
 
 before_script: git fetch --unshallow
-script: "dub --config=test"
+script:
+  - "dub --config=test"
+  - cd test_package && dub build
```

Comment:
I can't run `test.d` locally since Arch Linux isn't supported. Even if it were, I'd rather not download clang when I have it installed. Took a while to figure out `test.d` as well.